Code/Framework/AzCore/AzCore/std/hash_table.h

```diff
@@ -972,28 +987,32 @@ namespace AZStd
             rhs.clear();
         }
 
+        // find_insert_position sets `insertIter` to where the element should be inserted
```

Comment:
nit: remove \`\` - only maybe useful in API comments doxygen might pick up, just noise here otherwise